src/main/webapp/app/tutor-course-dashboard/tutor-participation-graph/progress-bar/progress-bar.component.html

```diff
@@ -0,0 +1,15 @@
+<div class="progress position-relative" [ngbTooltip]="'artemisApp.tutorCourseDashboard.evaluateStudentsComplaints' | translate">
```

Comment:
Tooltip needs to be passed in as an Input to the component since it is not always the same.